dashboard/src2/components/SiteUpdateDialog.vue

```diff
@@ -18,7 +18,7 @@
 							v-model="skipFailingPatches"
 						/>
 						<FormControl
-							label="Skip taking backup for this update (If the update fails, rollback will not be happen)"
+							label="Skip taking backup for this update (If the update fails, rollback will not happen)"
```

Comment:
```suggestion
							label="Skip taking backup for this update (If the update fails, rollback will not occur)"
```
(adding suggestion and committing, so I can avoid pulling your PR to my local)

Reply:
Also thanks for catching the mistake 😅